.github/workflows/request-reviews.yml

```diff
@@ -55,42 +61,64 @@ jobs:
           TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
           WORKSPACE_PATH: ${{ github.workspace }}
         run: |
-            import logging
+            import git

```

Comment:
Why isn't this in its own .py file?

Reply:
I didn't see the need since it only implements this workflow. If you'd prefer it to be in a file, I can do that.

Reply:
It is hard to maintain and test locally when in yml file.  I would prefer a .py file

Reply:
I moved it to its own Python file.